agent/helpers.go

```diff
@@ -128,10 +128,8 @@ func (h Helper) ensureRouteToEndpoint(netif *NetIf) error {
 		via := "dev"
 		dest := netif.Name
 
-		if err := h.createRoute(netif.IP,
-			mask, via, dest); err != nil {
-
-			// Or report error
+		err := h.createRoute(netif.IP, mask, via, dest, "src", h.Agent.networkConfig.romanaGW.String())
```

Comment:
Right here, ensureRouteToEndoint needs to know that (a) createRoute uses "ip" and (b) the exact syntax of "ip" for route creation.